packages/block-library/src/navigation-submenu/edit.js

```diff
@@ -394,6 +396,7 @@ export default function NavigationSubmenuEdit( {
 							rel: '',
 						} );
```

Comment:
I think it might be worth reconsidering what resetting some of these attributes means.

In a lot of cases, the block will have a post `id` (also `kind` and `type`), meaning it's linked to a particular post, and I think reset could set the value back to that post's title. Resetting to an empty string is quite destructive in that situation.

It'll take a bit of work as there's no code in the block at the moment to make that happen so it's something that can be handled in a separate PR/issue.

I think @getdave might be looking at some related issues at the moment.